Employees/Controllers/DeveloperController.cs

```diff
@@ -0,0 +1,31 @@
+using Employees.Models;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace Employees.Controllers
+{
+    [Route("api/developer")]
+    [Produces("application/json")]
+    [ApiController]
+    public class DeveloperController : ControllerBase
```

Comment:
no need for a separate controller for each type, add one employee controller.
remind me to discuss this further.